src/Sylius/Component/User/Security/Provider/PasswordResetTokenProvider.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Component\User\Security\Provider;
+
+use Sylius\Component\Resource\Repository\RepositoryInterface;
+use Sylius\Component\User\Security\Generator\GeneratorInterface;
+
+/**
+ * @author Jan Góralski <jan.goralski@lakion.com>
+ */
+class PasswordResetTokenProvider extends AbstractUniqueTokenProvider
```

Comment:
Finalize all the things! Extending this class will do no good, so let's enforce it :)